File: 2137-final-value-of-variable-after-performing-operations/final-value-of-variable-after-performing-operations.dart
class Solution {
  int finalValueAfterOperations(List<String> operations) {
    int x = 0;
    for (String op in operations) {
      if (op == "++X" || op == "X++") {
        x++;
      } else if (op == "--X" || op == "X--") {
        x--;
      }
    }
    return x;
  }
}

File: 2137-final-value-of-variable-after-performing-operations/final-value-of-variable-after-performing-operations.ts
function finalValueAfterOperations(operations: string[]): number {
    let X = 0;
    
    for (let operation of operations) {
        if (operation.includes("++")) {
            X += 1;
        } else if (operation.includes("--")) {
            X -= 1;
        }
    }
    
    return X;
}
